### PR TITLE
fix: (Core) Quick View title's corners and added ability to align title

### PR DESCRIPTION
**Pull request by olshansky**
**Note:** issue with title gaps resolved here https://github.com/SAP/fundamental-styles/pull/2069
#### Please provide a link to the associated issue.
Closes: https://github.com/SAP/fundamental-ngx/issues/4406
Related: https://github.com/SAP/fundamental-ngx/issues/4400 
#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist

Documentation checklist:
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

**Review by EvilAlexei — APPROVED** [review]

**Comment by puru-hk** [comment]
Tested these issues . it is working as expected.
#4406
#4400

**Review by Betrozov — APPROVED** [review]

**Comment by JKMarkowski** [comment]
Code looks good to me, but is this done on purpose?
image

**Comment by olshansky** [comment]
> @olshansky Code looks good to me, but is this(title - employee) done on purpose?

@JKMarkowski I've mentioned it here in the first line of PR's description 🙃
> **Note:** issue with title gaps resolved here https://github.com/SAP/fundamental-styles/pull/2069

**Comment by InnaAtanasova** [comment]
> > @olshansky Code looks good to me, but is this(title - employee) done on purpose?
> 
> @JKMarkowski I've mentioned it here in the first line of PR's description 🙃
> 
> > **Note:** issue with title gaps resolved here [SAP/fundamental-styles#2069](https://github.com/SAP/fundamental-styles/pull/2069)

Trust me, no one is reading the PR description :D saying this from experience :D

File: libs/core/src/lib/quick-view/quick-view-title/quick-view-title.component.ts [hunk 1]
```diff
@@ -1,8 +1,11 @@
...
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
 
 @Component({
     selector: 'fd-quick-view-title',
     templateUrl: './quick-view-title.component.html',
     changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class QuickViewTitleComponent {}
+export class QuickViewTitleComponent {
+    /** Alignment of title. */
```
Comment (InnaAtanasova) [inline libs/core/src/lib/quick-view/quick-view-title/quick-view-title.component.ts:9]: Could you please add to the typedocs the text:
Options include 'left', 'right' and 'middle'. The default is set to 'middle'.

Thank you!

**Review by InnaAtanasova — APPROVED** [review]

**Pull request merged**